cfgov/v1/atomic_elements/tables.py

```diff
@@ -114,6 +114,7 @@ class Table(blocks.StructBlock):
             ),
         ]
     )
+    caption = blocks.RichTextBlock(required=False)
```

Comment:
Can we restrict the options here, to prevent insertion of rich text elements that don't make sense in captions? I'm guessing we want bold, maybe italics, maybe the ability to link to pages or documents? See relevant Wagtail docs [here](https://docs.wagtail.org/en/stable/advanced_topics/customisation/page_editing_interface.html#rich-text-features).